src/components/Message/Message.tsx

```diff
@@ -200,7 +201,10 @@ export const Message = <
   const reactionSelectorRef = useRef<HTMLDivElement | null>(null);
 
   const handleAction = useActionHandler(message);
-  const handleDelete = useDeleteHandler(message);
+  const handleDelete = useDeleteHandler(message, {
+    getErrorNotification: getDeleteMessageErrorNotification,
+    notify: addNotification,
+  });
```

Comment:
move this down ahead of the flag handler and give it some space, these others up here are the inline ones